map-api-react/src/components/LayerMap.tsx

```diff
@@ -160,20 +160,31 @@ const LayerMap: React.FC<LayerMapProps> = () => {
         if (layer?.elements) {
           layer.elements.forEach((element: LayerElement) => {
             element.features.forEach((feature) => {
-              const icon = feature.style?.symbol?.symbol?.sidc
-                ? getMil2525Icon(feature.style.symbol.symbol.sidc)
-                : getMil2525Icon(null);
-
-              features.push({
-                type: 'Feature',
-                geometry: feature.geometry,
-                properties: {
-                  ...feature.style,
-                  id: element.id,
-                  label: element.label,
-                  icon,
-                },
-              });
+              const props = {
+                ...feature.style,
+                id: element.id,
+                label: element.label,
+                sidc: feature.style?.symbol?.symbol?.sidc,
+              }
+              if (feature.geometry.type == "Feature") {
+                features.push({
+                  ...feature.geometry,
+                  properties: props,
+              })
+              } else if (feature.geometry.type == "FeatureCollection") {
+                feature.geometry.features.forEach((innerFeature) => {
+                  features.push({
+                    ...innerFeature,
+                    properties: props,
+                  })
+                })
```

Comment:
This is a little funky - the Defense SDK actually flattens GeoJSON into `Feature | FeatureCollection | Point | MultiPoint | LineString | MultiLineString | Polygon | MultiPolygon | GeometryCollection`, while the GeoJSON spec is `Feature | FeatureCollection | Geometry` where Geometry is `Point | MultiPoint | LineString | MultiLineString | Polygon | MultiPolygon | GeometryCollection`.

I will raise this internally to make sure I'm understanding the issue correctly and potentially push forward a fix to make this consistent with IETF's definition.